airflow/providers/snowflake/transfers/copy_into_snowflake.py

```diff
@@ -15,8 +15,9 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-"""This module contains abstract operator that child classes implements
-COPY INTO <TABLE> SQL in Snowflake
+"""This module contains abstract operator that child classes implement.
+
+COPY INTO <TABLE> SQL in Snowflake.
 """
```

Comment:
This doesn’t looks right, I think the two lines are supposed to be one single sentence.